core/src/main/java/de/danoeh/antennapod/core/feed/Feed.java

```diff
@@ -390,12 +393,59 @@ public void setFeedIdentifier(String feedIdentifier) {
         this.feedIdentifier = feedIdentifier;
     }
 
-    public String getPaymentLink() {
-        return paymentLink;
+    public ArrayList<FeedFunding> extractPaymentLinks(String payLinks) {
```

Comment:
The `Feed` class already is pretty big. How about moving the serialization code to the `FeedFunding` class?